en/snap/platform/overview.md

```diff
@@ -341,24 +345,24 @@ Please complete the following steps:
 | Error Redirect URL | [your-site-url]/payment_ipn/veritrans/error |
 | Unfinish Redirect URL | [your-site-url]/payment_ipn/veritrans/unfinish |
 
-> **Note:**
+> ?>***Note:***
 >
 > Please make sure to input **http://** or **https://** when filling Notification URL and Redirect URL, according to your web-server configuration. 
 > 
 > If you are not sure, try opening your web URL in a browser, and check the URL is **http** or **https** on the address bar.
```

Comment:
Did you check the render result? please fix

<img width="781" alt="Screen Shot 2020-11-20 at 13 12 48" src="https://user-images.githubusercontent.com/13027142/99765667-1e33c300-2b32-11eb-8f1c-5d1225a2f978.png">